Issue on npm install

npm ERR! xz@1.0.3 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the xz@1.0.3 install script.
npm ERR! This is most likely a problem with the xz package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls xz
npm ERR! There is likely additional logging output above.
npm ERR! System Darwin 13.4.0

--- a/index.js
+++ b/index.js
@@ -181,7 +181,7 @@ var populate = function(that, file, header, engine) {
         debug('cluster is compressed? %s (%d)', compressed !== 0, compressed)
 
         var stream = file.createReadStream({start:cluster.offset+1, end:cluster.offset+1500000}) // haxx - TODO: fix me
-        var decomp = compressed < 2 ? through() : new (require('xz').Decompressor)
+        var decomp = compressed < 2 ? through() : new (require('lzma-native').Decompressor)
         var indexes = []
         var blobs = []
 

--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     "torrent-individual-piece-storage": "^3.0.0",
     "torrent-stream": "^0.15.0",
     "varint": "^3.0.1",
-    "xz": "^1.0.3"
+    "lzma-native": "^0.1.6"
   },
   "devDependencies": {},
   "bin": {

